operator/.osdk-scorecard.yaml

```diff
@@ -12,4 +12,4 @@ scorecard:
         namespace: "ibm-spectrum-scale-csi-driver"
         cr-manifest:
           - "deploy/crds/csiscaleoperators.csi.ibm.com_cr.yaml"
-        csv-path: "deploy/olm-catalog/ibm-spectrum-scale-csi-operator/2.0.0/ibm-spectrum-scale-csi-operator.v2.0.0.clusterserviceversion.yaml"
+        csv-path: "deploy/olm-catalog/ibm-spectrum-scale-csi-operator/2.0.1/ibm-spectrum-scale-csi-operator.v2.0.1.clusterserviceversion.yaml"
```

Comment:
Needs to be moved to 2.1.0 if that's the version we're using.

Reply:
Have switched back to 2.0.1. We will make the version changes after 2.0.1 is released.